tasks/store-publish/publishUi.ts

```diff
@@ -9,6 +9,7 @@ import fs = require('fs');
 import path = require('path');
 
 import tl = require('vsts-task-lib');
+var glob = require('glob');
```

Comment:
Since we do not have a typings for the version of glob module we need, I will keep this!